src/encoding_implementations/rectangular_binning.jl

```diff
@@ -160,7 +160,7 @@ function FixedRectangularBinning(b::RectangularBinning, x)
     T = eltype(x)
     ϵ = b.ϵ
     mini, maxi = minmaxima(x)
-    if ϵ isa Float64 || ϵ isa AbstractVector{<:AbstractFloat}
+    if ϵ isa <:AbstractFloat || ϵ isa AbstractVector{<:AbstractFloat}
```

Comment:
```suggestion
    if ϵ isa AbstractFloat || ϵ isa AbstractVector{<:AbstractFloat}
```
... `isa` does a type comparison. It's either `x isa Type` or `typeof(x) <: Type`. 

As you can see already online, the tests fail. When doing PRs, please do attempt to run the test suite locally before opening the PR. This way you can catch these things before the review process.

Reply:
yes sir